packages/fxa-event-broker/bin/worker.ts

```diff
@@ -34,11 +34,23 @@ async function main() {
     db
   );
   const pubsub = new PubSub();
+
+  // Extract region for SQS object
+  const serviceNotificationQueueUrl = Config.get('serviceNotificationQueueUrl');
+  const matchResult = serviceNotificationQueueUrl.match(/(?:.*\/sqs\.)([^\.]+)/i);
```

Comment:
Might be nice to wrap this in a little utility function and unit test it, but meh 🤷‍♂

Reply:
Good point, mainly cause its regex and it definitely took me some fiddling in a web console before it worked right. I'll file an issue to add that because it does feel precarious to let it hang out there like this.